server/node/package.json

```diff
@@ -14,15 +14,15 @@
   },
   "license": "Apache-2.0",
   "dependencies": {
-    "@paypal/paypal-server-sdk": "^0.6.1",
```

Comment:
Not anything to block this PR, but is there a place to get an abbreviated list of changes between these versions? [Comparing the two](https://github.com/paypal/PayPal-TypeScript-Server-SDK/compare/0.6.1...1.0.0), there's a _massive_ amount of changes.

Reply:
I don't think so unfortunately. I don't see a changelog in the server sdk repo.